content/pages/index.html

```diff
@@ -248,6 +248,29 @@ <h3 id="heading">Create IEEE 829 or free text Test Plan</h3>
                 changeDirection(direction);
                 setTimeout(function() { rollImage(direction); }, 50);
             });
+
+            /* Navigation */
+            var mobileBtnOpen = document.getElementsByClassName('open-menu')[0],
+                mobileBtnClose = document.getElementsByClassName('close-menu')[0],
+                mobileNav = document.getElementsByClassName('mobilenav')[0];
+                mobileLinks = document.querySelectorAll('.mobilenav a[href^="#"]');
```

Comment:
NOTE: after changing `mobilenav` class to an id these selectors will have to be updated